sir_cake/store/views.py

```diff
@@ -371,6 +375,7 @@ def _context_my_bag_total(request):
     return {
         'items_and_prices': items_and_prices,
         'total_price_all_items': total_price_all_items,
+        'preorder': preorder
```

Comment:
Why did you add this line? Just trying to understand where it is needed in the template?